### ISSUE TITLE
Update new subscriber email with final product icon

### ISSUE THREAD

**Issue by lmorchard** [mozilla/fxa#2027]
In PR #1791, we added support for an email to be sent to new subscribers with a product download link. It's currently using a placeholder image for the product. [We should update it with a real product icon.](https://github.com/mozilla/fxa/commit/7c84c152541de27fae5a405a973476f73994e6f9#diff-9437bdee47e1fcc021d46b2e851618f3R1766)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-241)
┆Issue Number: FXA-241

**Comment by johngruen** [comment]
Will add when icon is ready on 10/4

**Comment by lmorchard** [comment]
As a note to whomever takes this issue up - the icon is currently commented out but can be implemented here:

https://github.com/mozilla/fxa/blob/5a14f16af887ee227468f9ebb200663a336ad5a6/packages/fxa-auth-server/lib/senders/email.js#L1612-L1630

**Comment by johngruen** [comment]
Depends on #3451

**Comment by lmorchard** [comment]
This more depends on #3416 to figure out where / how to host icons - #3451 will display an image, but we need that icon hosted and its URL set in Stripe metadata as emailIconURL (which #3451 won't do)
